mobile/test/common/integration/base_client_integration_test.h

```diff
@@ -61,17 +62,24 @@ class BaseClientIntegrationTest : public BaseIntegrationTest {
   envoyToMobileHeaders(const Http::TestRequestHeaderMapImpl& request_headers);
   Event::ProvisionalDispatcherPtr dispatcher_ = std::make_unique<Event::ProvisionalDispatcher>();
   envoy_http_callbacks bridge_callbacks_;
-  ConditionalInitializer terminal_callback_;
-  callbacks_called cc_{0, 0, 0, 0, 0, 0, 0, "", &terminal_callback_, {}};
+  std::shared_ptr<ConditionalInitializer> terminal_callback_;
+  std::vector<std::shared_ptr<ConditionalInitializer>> multi_terminal_callbacks_;
+  std::shared_ptr<callbacks_called> cc_;
```

Comment:
same for here, this can be a reference, not a shared_ptr.  then, all the calls below can stay `cc_.` instead of `cc_->`.